metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/converters/HiveConnectorInfoConverter.java

```diff
@@ -70,7 +70,12 @@ public HiveConnectorInfoConverter(final HiveTypeConverter hiveTypeConverter) {
         this.hiveTypeConverter = hiveTypeConverter;
     }
 
-    private static Date epochSecondsToDate(final long seconds) {
+    /**
+     * Converts epoch time to Date.
+     * @param seconds time in seconds
+     * @return Date
+     */
+    public static Date epochSecondsToDate(final long seconds) {
```

Comment:
Any reason you don't just leave this as an `Instant` instead of returning a `Date`? `Date` is known to have lots of issues so it seems to me using `Instant` from the API is cleaner and if the consumer of the API really needs a `Date` it can deal with those issues locally

Reply:
We can look into this separately. This PR just exposed the method as a public method